echopype/convert/set_groups_ad2cp.py

```diff
@@ -11,6 +11,7 @@
 
 class SetGroupsAd2cp(SetGroupsBase):
     def write(self, ds: xr.Dataset, group: str):
+        ds.attrs["Units"] = "velocity: m/s, temperature: deg Celsius, pressure: dBar, direction: degrees, length: m, voltage: V, magnitude: dB, echosounder data: dB/count"
```

Comment:
Add this as memo from our meeting: Units are attributes for each of the data variables.